packages/components/atoms/f-form-field/src/assets/scss/common.scss

```diff
@@ -16,7 +16,7 @@ $form-input-borderColour                : f.$color-border-default;
 $form-input-borderRadius                : f.$radius-rounded-c;
 $form-input-borderColour--invalid       : f.$color-support-error;
 
-$form-input-focus--boxShadow            : 0 0 0 2px f.$color-focus;
+$form-input-focus--boxShadow            : 0 0 0 2px f.$color-focus-inner, 0 0 0 4px f.$color-focus-outer;
```

Comment:
Using `%u-elementFocus` here would mess up the positioning of the checkboxes, etc.